Makefile

```diff
@@ -61,5 +61,4 @@ disable_auto_delete:
 	$(eval export DISABLE_AUTODELETE=1)
 
 deploy_pipelines:
-	concourse/scripts/pipelines-bosh-cloudfoundry.sh
-	concourse/scripts/pipelines-failure-testing.sh
+	concourse/scripts/push-deployer-pipelines.sh
```

Comment:
I'd rather avoid creating another wrapper script if possible. I've made a simple attempt in branch fly_versions_2, isn't it good enough?